kyuubi-common/src/main/scala/org/apache/kyuubi/util/ArgsParser.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kyuubi.util
+
+import org.apache.kyuubi.config.KyuubiConf
+
+object ArgsParser {
+
+  def fromCommandLineArgs(args: Array[String], conf: KyuubiConf): Unit = {
+    for (i <- args.indices by 2) {
+      require(args(i) == "--conf", s"Unrecognized main arguments prefix ${args(i)}")
+      require(i + 1 <= args.length, "Command arguments ended unexpectedly")
```

Comment:
maybe we can check this once

Reply:
done

Reply:
No in this way, I mean checking out of the loop

Reply:
done